rocAL/rocAL_pybind/rali_pybind.cpp

```diff
@@ -235,6 +235,10 @@ namespace rali{
             .value("NHWC",RALI_NHWC)
```

Comment:
looks like the above examples for training was deleted without converting to new API. We need to have them ported as well

Reply:
the file needs to be renamed to rocal_pybind.cpp

Reply:
Have added this change as a part of rocal restructuring development branch of ours